pkg/container-disk/container-disk_test.go

```diff
@@ -92,78 +91,98 @@ var _ = Describe("ContainerDisk", func() {
 				Entry("raw disk", "raw"),
 			)
 			It("by verifying error when no disk is present", func() {
-
-				vmi := api.NewMinimalVMI("fake-vmi")
-				appendContainerDisk(vmi, "r0")
+				vmi := libvmi.New(
+					libvmi.WithContainerDisk("r0", "someimage:v1.2.3.4"),
+				)
+				By("Ensuring no disk is present")
+				// Don't create any disk file to simulate the condition
+
+				By("Trying to get the volume mount directory on guest and expecting an error")
+				volumeMountDir := GetVolumeMountDirOnGuest(vmi)
+				_, err := os.Stat(filepath.Join(volumeMountDir, "disk_0.img"))
+				Expect(err).To(HaveOccurred())
+				Expect(errors.Is(err, os.ErrNotExist)).To(BeTrue())
```

Comment:
This was missing before, correct?

Reply:
correct 
It was just and i did not see the assertion
```go
It("by verifying error when no disk is present", func() {
vmi := api.NewMinimalVMI("fake-vmi")
appendContainerDisk(vmi, "r0")
})
```
i just add extra assertion , just to make sure everything is working fine and make sense but if you see no need i can remove it
```go
It("by verifying error when no disk is present", func() {
vmi := libvmi.New()

By("Trying to get the volume mount directory on guest and expecting an error")
volumeMountDir := GetVolumeMountDirOnGuest(vmi)
_, err := os.Stat(filepath.Join(volumeMountDir, "disk_0.img"))
Expect(err).To(HaveOccurred())
Expect(errors.Is(err, os.ErrNotExist)).To(BeTrue())
})

what do you think ??

Reply:
I think here would be more indicate to use `MatchError` -> `Expect(err).To(MatchError(os.ErrNotExist))`